hazen.py

```diff
@@ -11,37 +11,64 @@
 __author__ = "mohammad_haris.shuaib@kcl.ac.uk"
 
 
+import os
+import inspect
+
+import os
+import inspect
+import ast
+
+
 def register_tasks_in_db():
     from hazenlib import tasks as hazen_tasks
-    # TODO: Change to read from hazenlib classes
-    tasks = {f'{modname}': importlib.import_module(
-        f'hazenlib.tasks.{modname}') for importer, modname, ispkg in
-        pkgutil.iter_modules(hazen_tasks.__path__)}
+    tasks = {
+        f'{modname}': importlib.import_module(f'hazenlib.tasks.{modname}')
+        for importer, modname, ispkg in pkgutil.iter_modules(hazen_tasks.__path__)
+    }
+
+    def get_module_docstring(filepath):
+        with open(filepath, 'r') as f:
+            content = f.read()
+
+        parsed_module = ast.parse(content)
+        for node in parsed_module.body:
+            if isinstance(node, ast.Expr) and isinstance(node.value, ast.Str):
+                return inspect.cleandoc(node.value.s)
+        return ''
 
     with app.app_context():
         stored_tasks = Task.query.all()
 
         for stored_task in stored_tasks:
             if stored_task.name in tasks.keys():
+                module_file = os.path.join(os.path.dirname(hazen_tasks.__file__), f"{stored_task.name}.py")
+                docstring = get_module_docstring(module_file)
+
+                # Update the task's docstring
+                stored_task.docstring = docstring
+
+                # Commit the changes to the database
+                db.session.commit()
```

Comment:
@Lucrezia-Cester I think the issue I'm having is that this line doesn't seem to be committing the docstrings to my `Tasks` table.